apps/user-office-frontend/src/components/questionary/questionaryComponents/NumberInput/QuestionNumberInputForm.tsx

```diff
@@ -81,24 +93,37 @@ export const QuestionNumberForm: FC<QuestionFormProps> = (props) => {
               }}
               InputProps={{ 'data-cy': 'required' }}
             />
+            <FormControl fullWidth>
+              <Autocomplete
+                id="config-units"
+                multiple
+                options={units}
+                getOptionLabel={({ unit, symbol, quantity }) =>
+                  `${symbol} (${unit}) - ${quantity}`
+                }
+                renderInput={(params) => (
+                  <MaterialTextField {...params} label="Units" margin="none" />
+                )}
+                onChange={(_event, newValue) => {
+                  setSelectedUnits(newValue);
+                  setFieldValue('config.units', newValue);
+                }}
+                value={selectedUnits ?? undefined}
+                data-cy="units"
+              />
 
-            <Autocomplete
-              id="config-units"
-              multiple
-              options={units}
-              getOptionLabel={({ unit, symbol, quantity }) =>
-                `${symbol} (${unit}) - ${quantity}`
-              }
-              renderInput={(params) => (
-                <MaterialTextField {...params} label="Units" margin="none" />
-              )}
-              onChange={(_event, newValue) => {
-                setSelectedUnits(newValue);
-                setFieldValue('config.units', newValue);
-              }}
-              value={selectedUnits ?? undefined}
-              data-cy="units"
-            />
+              <Link
+                href="/Units/"
+                target="_blank"
+                style={{ textAlign: 'right' }}
```

Comment:
Maybe you can extract this style in its own class instead of using styles. The same thing that you did with the icon. That way we keep the styling of the whole component in one place.

Reply:
Glad you mentioned it. I was a bit unsure whether should I use inline style, since its a small change. It definitely makes sense to put all stylings in one place.